metabase/metabase
# Search Milestone 2: Add `created_by` search filter

`created_by` filter for Search Milestone 2: #27982 

### Description

Adds `created_by` filter to the search sidebar. The code uses `useUserListQuery` to populate the list, and presents a "No users found" if the search yields no results.


### How to verify

Go to the search sidebar, and try to filter by user. Only assets created by that user should be available in the search.

After filtering, try to refresh the page or paste the URL into a new window to test that hydrating the search filters still works when entering the search directly from a URL. The results should remain the same and should behave in the same way.

### Tests
